app/lib/screens/history/history.dart

```diff
@@ -11,6 +18,36 @@ class _TravelHistoryState extends State<TravelHistory> {
     print('Init: Travel history');
   }
 
+  final pdf = pw.Document(); //Create a blank document
```

Comment:
As per our previous discussion we should extract this into its own file, but I think it would go well in `pdf_preview.dart`.

- [ ] Move out of `history.dart`

Reply:
@holtzmak I moved this into pdfpreview.dart file. Thanks.